libraries/chain/include/graphene/chain/protocol/htlc.hpp

```diff
@@ -126,16 +126,16 @@ namespace graphene {
          struct fee_parameters_type {};
 
          htlc_redeemed_operation() {}
-         htlc_redeemed_operation( htlc_id_type htlc_id, account_id_type from, account_id_type to, asset amount) :
-               htlc_id(htlc_id), from(from), to(to), amount(amount) {}
+         htlc_redeemed_operation( htlc_id_type htlc_id, account_id_type from, account_id_type to, account_id_type redeemer, asset amount ) :
+               htlc_id(htlc_id), from(from), to(to), redeemer(redeemer), amount(amount) {}
 
          account_id_type fee_payer()const { return to; }
          void validate()const { FC_ASSERT( !"virtual operation" ); }
 
          share_type      calculate_fee(const fee_parameters_type& k)const { return 0; }
 
          htlc_id_type htlc_id;
-         account_id_type from, to;
+         account_id_type from, to, redeemer;
```

Comment:
I think we need to bump DB_VERSION if to add a field here.

Reply:
Done.

Reply:
I disagree. We don't have any such operations in the database yet, therefore the change has no effect. We can avoid the replay.

Reply:
We can do the bump for testnet branch only.

Reply:
Done